src/database.ts

```diff
@@ -69,13 +79,15 @@ export class Database {
     try {
       // get connection to check
       try {
-        connection = await this.sequelize.connectionManager.getConnection({type: 'write'});
+        connection = await this.sequelize.connectionManager.getConnection({
+          type: 'write',
+        });
       } catch (e) {
         throw new Error('Database connection not available!');
       }
 
       // validate connection
-      if (!((this.sequelize.connectionManager as any).validate(connection))) {
+      if (!(this.sequelize.connectionManager as any).validate(connection)) {
```

Comment:
I see in types that there is `this.sequilize.validate` but you are calling on `connectionManager`
Is `sequilize.validate` not working?

Reply:
If it's not your code and you didn't check why, then don't bother with my Qs

Reply:
I actually ran prettier countless of times and excluded that file from git index till I forgot  ^^'